packages/app/src/debug/DebugPageHeader.vue

```diff
@@ -147,7 +149,14 @@ const debug = computed(() => props.gql)
 
 const relativeCreatedAt = computed(() => dayjs(new Date(debug.value.createdAt!)).fromNow())
 
-const totalDuration = useDurationFormat(debug.value.totalDuration ?? 0)
+/*
+  Format duration to in HH[h] mm[m] ss[s] format. The `totalDuration` field is milliseconds. Remove the leading "00h" if the value is less
+  than an hour. Currently, there is no expectation that a run duration will be greater 24 hours or greater, so it is okay that
+  this format would "roll-over" in that scenario.
+  Ex: 1 second which is 1000ms = 00m 01s
+  Ex: 1 hour and 1 second which is 3601000ms = 01h 00m 01s
+*/
+const totalDuration = computed(() => dayjs.duration(debug.value.totalDuration ?? 0).format('HH[h] mm[m] ss[s]').replace(/^0+h /, ''))
```

Comment:
Instead of changing `useDurationFormat`, I added this here so that it doesn't change the runs page or the duration on the spec breakdown

Reply:
Test for this would be good 🙏 

Also could include one for >10h, and also >100h (unrealistic, but what happens then?)

Reply:
Agree we should have a test in `DebugPageHeader.cy.tsx` to run through all the combinations

Reply:
If the duration goes over 24 hours the format here will break. We're assuming that the duration will be less than 24 hours. You can see that in the component tests for `useDurationFormat` https://github.com/cypress-io/cypress/blob/21b576b95afa65789f654e969014519f13a75b10/packages/app/src/composables/useDurationFormat.cy.tsx#L10-L11